apps/test/unit/redux/unitSelectionReduxTest.js

```diff
@@ -1,82 +1,39 @@
 import {assert} from '../../util/reconfiguredChai';
 import unitSelection, {
-  setValidScripts,
   setScriptId,
   getSelectedScriptName,
-  getSelectedScriptDescription
+  getSelectedScriptDescription,
+  setCourseVersionsWithProgress
 } from '@cdo/apps/redux/unitSelectionRedux';
-
-const fakeValidScripts = [
-  {
-    category: 'category1',
-    category_priority: 1,
-    id: 456,
-    name: 'Script Name',
-    position: 23,
-    description: 'Description of Script'
-  },
-  {
-    category: 'csp',
-    category_priority: 1,
-    id: 300,
-    name: 'csp1',
-    position: 23,
-    description: 'CSP Unit 1'
-  },
-  {
-    // Use a different category to make sure we aren't relying on it to group
-    // units within courses.
-    category: 'other csp',
-    category_priority: 1,
-    id: 301,
-    name: 'csp2',
-    position: 23,
-    description: 'CSP Unit 2'
-  },
-  // Include Express Course to use as default if needed
-  {
-    category: 'CS Fundamentals',
-    category_priority: 1,
-    id: 182,
-    name: 'Corso Rapido',
-    position: 6,
-    script_name: 'express-2017',
-    description: 'CSF Spanish Course'
-  }
-];
-
-const fakeValidCourses = [
-  {
-    id: 99,
-    script_ids: [300, 301]
-  }
-];
+import {fakeCourseVersionsWithProgress} from '@cdo/apps/templates/teacherDashboard/teacherDashboardTestHelpers';
 
 describe('unitSelectionRedux', () => {
   const initialState = unitSelection(undefined, {});
 
+  it('if no scriptId then defaults to first unit of first course version', () => {
+    const action = setCourseVersionsWithProgress(
+      fakeCourseVersionsWithProgress
+    );
+    const nextState = unitSelection(initialState, action);
+    assert.deepEqual(nextState.scriptId, '1');
```

Comment:
I agree with you that using the fake data here is unclear. I was working to match the existing pattern that was already being used in this file. I think with the timeline of this work I'm going to hold off on trying to update this right now but agree it would be a good thing to come back to.

Reply:
Could you open a ticket on either EPTools or LP (wherever you think this more appropriately belongs) to return to this?

Reply:
https://codedotorg.atlassian.net/browse/LP-2325

Reply:
Thanks!